x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/ResultSetTestCase.java

```diff
@@ -5,6 +5,7 @@
  */
 package org.elasticsearch.xpack.sql.qa.jdbc;
 
+import com.carrotsearch.randomizedtesting.annotations.Repeat;
```

Comment:
This doesn't seem to be used anywhere...

Reply:
Yep, thx, leftover, removed it already